src/api/app/views/webui/request/_request_history.html.haml

```diff
@@ -1,7 +1,8 @@
-.media
```

Comment:
I only looked at the changes in this view, but I would assume they are all the same since the same pattern was applied over and over again.

In the new version, the gap between the image and the text is bigger. See below.

Before:
![before](https://user-images.githubusercontent.com/1102934/211281787-267f344f-c00c-4179-a2ba-f99a645ac068.png)

Now:
![after](https://user-images.githubusercontent.com/1102934/211281781-6d103b8b-1753-49c2-8e88-39a0f3efe482.png)